common/meta/grpc/src/grpc_client.rs

```diff
@@ -775,10 +848,24 @@ impl MetaGrpcClient {
 
         Ok(reply)
     }
+    async fn prepare_retry_req(&self) {
+        {
+            let ca = self.current_endpoint.lock().unwrap();
+            let mut ue = self.unhealthy_endpoints.lock().unwrap();
+            ue.insert((*ca).as_ref().unwrap().clone(), ());
+        }
+        {
+            let mut token = self.token.write().await;
+            *token = None;
+        }
+    }
```

Comment:
Naming a method with what it does would be better than with what it does for.

E.g., to me, (maybe unthoughtful name) `mark_as_unhealthy()` would be better than `preapre_retry_req()`